tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -3943,4 +3943,226 @@ public function provide_test_is_valid_optimized_checkout_page() {
+		$order_helper->update_stripe_checkout_session_id( $order, $checkout_session_id );
+		$order_helper->update_stripe_presentment_amount( $order, 1500 );
+		$order_helper->update_stripe_presentment_currency( $order, 'eur' );
+
+		// No checkout session is cached — if the code tries to fetch it, the test will fail.
+		$formatted_total = '$10.00';
+		$result          = $this->mock_gateway->add_converted_currency_information( $formatted_total, $order );
+
+		$expected_amount = WC_Stripe_Helper::get_woocommerce_amount_from_stripe_amount( 1500, 'eur' );
+
+		$this->assertEquals( '$10.00 (' . $expected_amount . ' EUR)', $result );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_converted_currency_information_reads_from_order_meta_without_api_call is not named in camelCase. (undefined)

(CamelCaseMethodName)